libs/cln-version-manager/clnvm/cln_version_manager.py

```diff
@@ -67,7 +67,7 @@ class VersionDescriptor:
     VersionDescriptor(
         tag="v23.08gl1",
         url="https://storage.googleapis.com/greenlight-artifacts/cln/lightningd-v23.08gl1.tar.bz2",
-        checksum="700e2f4765ba2f8d83ea26e0dc5b47c7176b665c7d6945bdfed2207c90ed9ebd",
+        checksum="724a5ae70e1e5fda67751a37dbb56648735b6d55394dcfe2193f15892c483d55",
```

Comment:
I'm sorry, this hash has changed again, now that I backported CLN#7190. Would it make sense for us to publish the hashes to GCS and fetch them from there, rather than hard-coding them? We could also make the version descriptor themselves loadable from the URL.

Reply:
Good questions, I don't know what the best solution is.

When writing the code I assumed we would never change hashes upstream. 
This assumption has proven to be false. Hosting an index on GCS seems like a good idea.
It also helps us to know when we should update dependencies locally.

I can prepare such a file and the tooling to maintain it.